canvas_modules/harness/cypress/support/canvas/toolbar-cmds.js

```diff
@@ -74,3 +74,21 @@ Cypress.Commands.add("clickToolbarZoomToFit", () => {
 Cypress.Commands.add("clickToolbarNotifications", () => {
 	cy.get("#notificationCounterIcon-action").click();
 });
+
+Cypress.Commands.add("shortcutKeysCut", () => {
+	// Press Ctrl/Cmnd+x to Cut
+	const keySelector = Cypress.platform === "darwin" ? "{meta}" : "{ctrl}";
+	cy.get("body").type(keySelector + "{x}", { release: false });
+});
+
+Cypress.Commands.add("shortcutKeysCopy", () => {
+	// Press Ctrl/Cmnd+c to Copy
+	const keySelector = Cypress.platform === "darwin" ? "{meta}" : "{ctrl}";
+	cy.get("body").type(keySelector + "{c}", { release: false });
+});
+
+Cypress.Commands.add("shortcutKeysPaste", () => {
+	// Press Ctrl/Cmnd+v to Paste
+	const keySelector = Cypress.platform === "darwin" ? "{meta}" : "{ctrl}";
```

Comment:
Just a minor thing it seems a shame to have this line repeated through the code -- especially since it is such a strange syntax ('darwin' ??) so maybe this line should be encapsulated in a single function.